lib/bolt/pal.rb

```diff
@@ -521,10 +521,28 @@ def get_plan_mtime(path)
       end
     end
 
-    def convert_plan(plan_path)
+    def convert_plan(plan)
+      # If the path doesn't exist, check if it's a plan name
+      unless File.exist?(plan)
```

Comment:
It looks like we immediately expand this path in the `transpile` method and then don't reuse the relative path - I'd definitely sleep a little better if we just expand it here before checking if the file exists and then pass the absolute path to the transpile method (whether it's from a path or plan name)